src/app/about/about.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AboutComponent } from './about.component';
```

Comment:
Let's generate all components under **component** directory to better organize things.
All components which we will create to render content under about section, will be under component/about/ folder.

Going forward, we will have **service** folder under which, we will be generating services classes.